src/command_modules/azure-cli-monitor/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 0.1.2
 +++++
 * Minor fixes.
+* Fix issue #5151
```

Comment:
We can jettison "Minor fixes" if we have a real fix. Also, this should say exactly what was fixed instead of referencing a Github issue no one will look up.  `monitor log-profiles create`: Fixed "blah cannot be None" error.